code/__DEFINES/dcs/signals/signals_global.dm

```diff
@@ -68,3 +68,5 @@
 
 /// Global signal sent when a light mechanism is completed (try_id)
 #define COMSIG_GLOB_LIGHT_MECHANISM_COMPLETED "!light_mechanism_completed"
+///Global Signal sent when the crew wins the revolution (No arguments).
+#define COMSIG_GLOB_REVOLUTION_TAX_REMOVAL "!revs_tax_removal"
```

Comment:
```suggestion
#define COMSIG_GLOB_REVOLUTION_TAX_REMOVAL "!revolution_tax_removal"
```
The text of a comsig define should match the define name, to prevent weird collisions